### PR TITLE
Update sphinx theme to include edit link [GEOS-9282]

### PR DESCRIPTION
**Pull request by jodygarnett**
Update the sphinx theme to include an edit button

## Checklist

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [ x You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):

- [x] https://osgeo-org.atlassian.net/browse/GEOS-9282
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)

Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.

### CODE REVIEW

**Comment by jodygarnett** [comment]
Considering the above failure is in `GeoServer Security Tests Module` and this is a change to docs I would like to proceed with this change.

@ianturton are you available to review please?

File: doc/en/user/source/conf.py [hunk 1]
```diff
@@ -215,17 +230,16 @@
   'fncychap': '\\usepackage[Sonny]{fncychap}',
 'preamble': #"""\\usepackage[parfill]{parskip}
   """
-	\\hypersetup{
-		colorlinks = true,
+    \\hypersetup{
```
Comment (tbarsballe) [inline doc/en/user/source/conf.py]: Is this comment, or just weird syntax? If its just a comment, can it be removed?

Comment (jodygarnett) [reply doc/en/user/source/conf.py]: tabs vs spaces

**Comment by jodygarnett** [comment]
Consider back porting? Nope only for master ...

**Pull request merged**